### ISSUE TITLE
<img> should have alt attributes

### ISSUE THREAD

**Issue by h3xar0n** [tsturner24/prj-rev-bwfs-dasmoto#2]
Any img that is not window dressing should have an `alt` tag for the visually impaired. They use screen readers, and you can convey the main point of the image by using this attribute. For example, if you have a picture of a cup filled with brushes, brushcup.jpg, you could write it like so:

`<img src="./images/brushcup.jpg" alt="A cup filled with brushes">`

What do alt descriptions do you think you could provide for the following:

https://github.com/tsturner24/prj-rev-bwfs-dasmoto/blob/1894fa6383302e1d0e5eedbd8564e887e2fc7a8c/Arts%26Crafts/index.html#L12

https://github.com/tsturner24/prj-rev-bwfs-dasmoto/blob/1894fa6383302e1d0e5eedbd8564e887e2fc7a8c/Arts%26Crafts/index.html#L20

https://github.com/tsturner24/prj-rev-bwfs-dasmoto/blob/1894fa6383302e1d0e5eedbd8564e887e2fc7a8c/Arts%26Crafts/index.html#L28
